libcxx/include/__math/gamma.h

```diff
@@ -55,6 +55,31 @@ inline _LIBCPP_HIDE_FROM_ABI double tgamma(_A1 __x) _NOEXCEPT {
   return __builtin_tgamma((double)__x);
 }
 
+// __lgamma_r
+
+struct __lgamma_result {
+  double __result;
+  int __sign;
+};
```

Comment:
We're never using the `__sign`. Should we remove that part? @lntue do you think this makes any difference in the implementation?